### PR TITLE
Feature/simple evaluations UI part 2

### PR DESCRIPTION
**Pull request by geclos**

### CODE REVIEW

File: apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+            <Input
+              name='maxValueDescription'
+              defaultValue={resultConfiguration.maxValueDescription ?? ''}
+              placeholder='Description of the max value'
+            />
+          </FormField>
+        </FormFieldGroup>
+        <div className='flex flex-col gap-2 flex-grow'>
+          <Label>Additional instructions</Label>
+          <textarea
```
Comment (andresgutgon) [inline apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx:198]: We have a nice auto grow textarea atom
https://github.com/latitude-dev/latitude-llm/blob/main/packages/web-ui/src/ds/atoms/TextArea/index.tsx

File: apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/TextEvaluationEditor/index.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        <FormField label='Value description'>
+          <Input
+            name='valueDescription'
+            defaultValue={resultConfiguration.valueDescription ?? ''}
+            placeholder='Description of the evaluation output value'
+          />
+        </FormField>
+        <div className='flex flex-col gap-2 flex-grow'>
+          <Label>Additional instructions</Label>
+          <textarea
```
Comment (andresgutgon) [inline apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/TextEvaluationEditor/index.tsx:169]: Ditto

File: apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx (hunk 1 above)
Comment (geclos) [reply apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx:198]: yeah i tried that component wasn't working out for me, it just wasn't taking the full height so I'm not sure what it's supposed to do?

Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx:198]: Fix it no?

Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx:198]: Also I noticed you do `<FormField><Input /></FormField>`. No need just pass `label` to `<Input />`. It has the form field included

Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx:198]: Let's comment over here
https://github.com/latitude-dev/latitude-llm/pull/578

**Pull request merged**